views/ok.ejs

```diff
@@ -1,35 +0,0 @@
-<html>
-<head>
-    <meta charset="utf-8">
-    <link href="bower_components/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
-    <script type="text/javascript" src="bower_components/jquery/dist/jquery.min.js" charset="utf-8"></script>
-
-</head>
-<body>
-<script type="text/javascript" src="http://localhost:3000/bundle.js" charset="utf-8">
-</script>
-<script>
-    var ludwig = new Ludwig();
-</script>
-
-</a>
-</body>
-
-<body>
-
-<!-- Begin page content -->
-<div class="container">
-    <h1>Merci!</h1>
-    <p>Votre suggestion a bien été enregistrée, elle est accessible en suivant <a href="<%=pullRequestURL%>">ce lien</a></p>
```

Comment:
Why remove this view? We're going to need to proxy GitHub's GUI too if we want to get non-tech contributors on board.

I understand you might want to make the API itself leaner, but I'm not sure this is worth removing the GUI altogether.